docs/UserGuide.md

```diff
@@ -2,1012 +2,1171 @@
+[Back to list of features](#features)
 
 ### Sorting list of students: `sort`
 
-Sorts list of students in student roster.
+You can sort your students either by their names or their student ID.
+
+**Command Format:**
 
-Format:
 ```
 sort <CATEGORY>
 ```
-
-- Current possible categories are `name` and `id`
+- `<CATEGORY>` is either `name` or `id`
 
 Examples:
-- `sort name` will sort list of students alphabetically according to name.
-- `sort id` will sort list of students alphanumerically according to student ID.
 
-![image](./images/UiStudentSortName.png)
-###### Figure 7
-![image](./images/UiStudentSortId.png)
-###### Figure 8
+- `sort name` will sort your list of students alphabetically according to name.
+- `sort id` will sort your list of students alphanumerically according to student ID.
+
+<img src="./images/UiStudentSortName.png" width="650" height="529"/>
+<br>
+<img src="./images/UiStudentSortId.png" width="650" height="529"/>
+<br>
 
-[Back to list of commands](#commands)
+[Back to list of features](#features)
 
-### Displays a student's information: `select`
+### Display a student's information: `select`
 
-![image](./images/UiStudentView.png)
-###### Figure 9
+This command allows you to select a student and display their full particulars.
+The selected student will be shown like this:
 
-Select a particular student and display his / her particulars.
+<img src="./images/UiStudentView.png" width="650" height="529"/>
+
+**Command Format:**
 
-Format:
 ```
 select <INDEX>
 ```
 
 Examples:
-- `select 1` will select the 1st student in the student list and display his / her particulars on the view panel on the right.
 
-[Back to list of commands](#commands)
+- `select 1` will select the first student in the student list and display his / her particulars on the view panel on the right.
+
+[Back to list of features](#features)
 
 ### Adding or deleting a remark: `remark`
 
-Format for adding / editing a remark:
+You can use this command to add, edit or remove remarks from a student.
+
+**Command Format for Adding/Editing a remark:**
+
 ```
 remark <INDEX> [<REMARK>]
 ```
 
-- Adds a remark to student in the student list whose index is specified in the `<INDEX>` field.
-- If there is already an existing remark, this command will replace current remark with `<REMARK>`.
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:** If there is already an existing remark, it will be replaced with the new remark.
+</div>
+
+
+**Command Format for deleting a remark:**
 
-Format for deleting a remark:
 ```
 remark <INDEX>
 ```
 
-- Deletes a remark from a student in the student list whose index is specified in the `<INDEX>` field.
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+  Accidentally deleted a remark? Don't worry, Try using the <code>undo</code> command!
+</div>
+<br>
 
 Examples:
+
 - `remark 1 Loves watching Sci-Fi movies` will add a remark of 'Loves watching Sci-Fi movies' to the first student
```

Comment:
```suggestion
- `remark 1 Loves watching Sci-Fi movies` will add a remark of "Loves watching Sci-Fi movies" to the first student
```

Just a minor one for consistency with updated document.

Reply:
thanks, fixed